### ISSUE TITLE
Old version of Github Logs Serverless app hardcoded in code

### ISSUE THREAD

**Issue by biffgaut** [cdklabs/aws-delivlib#1820]
A few months back I opened [Issue 1719](https://github.com/cdklabs/aws-delivlib/issues/1719),  along with a [PR that fixed the issue](https://github.com/cdklabs/aws-delivlib/pull/1721). At that time the PR was marked do not merge, with the comment:

> Hi @biffgaut, thank you for the contribution! Unfortunately we're not accepting any changes at this time while we're working on setting up additional testing for delivlib. We will get back to this PR once that is in place.

Eventually the PR was automatically closed due to inactivity and the Issue was also automatically closed as complete. But the bug is still there. (the other change in my PR, updating the JSII image, was addressed by someone else in a subsequent PR).

I'm creating this issue to again draw attention to this problem, and will be opening another PR to address it.

**Comment by biffgaut** [comment]
Until this gets fixed, if you are here because you're facing the same problem, here is the workaround we are using:

```
    const yourBuild = new AutoBuild(this, 'id', {
      // All your properties
    });
    // Use escape hatch to override the hardcoded version
    const logs = yourBuild.node.findChild('GitHubCodeBuildLogsSAR') as serverless.CfnApplication;
    logs.addPropertyOverride('Location.SemanticVersion', '1.6.0');
```

**Comment by biffgaut** [comment]
Here is the PR that addresses it - https://github.com/cdklabs/aws-delivlib/pull/1821

**Comment by moelasmar** [comment]
Thanks @biffgaut for raising this issue, and for the fix. The Pr got merged, I am closing this issue.
